content/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent.md

```diff
@@ -40,7 +40,7 @@ If you are a site administrator for {% data variables.location.product_location
 {% data reusables.ssh.key-type-support %}
 
 {% data reusables.command_line.open_the_multi_os_terminal %}
-1. Paste the text below, substituting in your {% data variables.product.product_name %} email address.
+1. Paste the text below, substituting the email with your {% data variables.product.product_name %} email address.
```

Comment:
Improving your suggestion, and using the verb `to replace` instead as it's more commonly used and understood.

```suggestion
1. Paste the text below, replacing the email used in the example with your {% data variables.product.product_name %} email address.
```